plugins/woocommerce/includes/gateways/paypal/includes/class-wc-gateway-paypal-constants.php

```diff
@@ -92,4 +92,36 @@ class WC_Gateway_Paypal_Constants {
 		'phone_number',
 		'national_number',
 	);
+
+	/**
+	 * List of currencies supported by PayPal.
+	 *
+	 * @var array<string>
+	 */
+	const SUPPORTED_CURRENCIES = array(
+		'AUD', // Australian Dollar.
+		'BRL', // Brazilian Real.
+		'CAD', // Canadian Dollar.
+		'CNY', // Chinese Renminbi.
+		'CZK', // Czech Koruna.
+		'DKK', // Danish Krone.
+		'EUR', // Euro.
+		'HKD', // Hong Kong Dollar.
+		'HUF', // Hungarian Forint.
+		'ILS', // Israeli New Shekel.
+		'JPY', // Japanese Yen.
+		'MYR', // Malaysian Ringgit.
+		'MXN', // Mexican Peso.
+		'TWD', // New Taiwan Dollar.
+		'NZD', // New Zealand Dollar.
+		'NOK', // Norwegian Krone.
+		'PHP', // Philippine Peso.
+		'PLN', // Polish Złoty.
+		'GBP', // Pound Sterling.
+		'SGD', // Singapore Dollar.
+		'SEK', // Swedish Krona.
+		'CHF', // Swiss Franc.
+		'THB', // Thai Baht.
+		'USD', // United States Dollar.
```

Comment:
@wjrosa we may need to add `RUB`. Details in p1763444033927989/1762953165.370549-slack-C100MSW86